README.md

```diff
@@ -1,26 +1,22 @@
 # SuperBenchmark
 
+[![MIT licensed](https://img.shields.io/badge/license-MIT-brightgreen.svg)](LICENSE)
 [![Lint](https://github.com/microsoft/superbenchmark/workflows/Lint/badge.svg)](https://github.com/microsoft/superbenchmark/actions?query=workflow%3ALint)
 [![Codecov](https://codecov.io/gh/microsoft/superbenchmark/branch/main/graph/badge.svg?token=DDiDLW7pSd)](https://codecov.io/gh/microsoft/superbenchmark)
 
-| Azure Pipelines | Build Status |
-| :---: | :---: |
-| cpu-unit-test | [![Build Status](https://dev.azure.com/msrasrg/SuperBenchmark/_apis/build/status/microsoft.superbenchmark?branchName=main)](https://dev.azure.com/msrasrg/SuperBenchmark/_build/latest?definitionId=77&branchName=main) |
-| gpu-unit-test | [![Build Status](https://dev.azure.com/msrasrg/SuperBenchmark/_apis/build/status/cuda-unit-test?branchName=main)](https://dev.azure.com/msrasrg/SuperBenchmark/_build/latest?definitionId=80&branchName=main) |
 
+**SuperBench** is a validation and profiling tool for AI infrastructure, which supports:
 
-SuperBench is a benchmarking and diagnosis tool for AI infrastructure,
-which supports:
-* Comprehensive AI infrastructure validation
+* AI infrastructure validation and diagnosis
     * Distributed validation tools to validate hundreds or thousands of servers automatically
     * Consider both raw hardware and E2E model performance with ML workload patterns
-    * Provide a fast and accurate way to detect and locate hardware problems
-    * Performance/Quality Gates for hardware and system release
-* Benchmarking with typical AI workload patterns
+    * Build a contract to identify hardware issues
+    * Provide infrastructural-oriented criteria as Performance/Quality Gates for hardware and system release
+    * Provide detailed performance report and advanced analysis tool  
+  
+* AI workload benchmarking and profiling
     * Provide comprehensive performance comparison between different existing hardware
-    * Give a better understanding for new DL software & hardware
-* Detailed performance analysis and diagnosis
-    * Provide detailed performance report and advanced analysis tool   
+    * Provide insights for hardware and software co-design
```

Comment:
not fit here?

Reply:
It is our vision in a full picture. So I think it is fine now to put it here.